test/CadlRanchProjects/type/union/src/Generated/EnumsOnly.cs

```diff
@@ -46,20 +46,20 @@ internal EnumsOnly(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, U
 
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <include file="Docs/EnumsOnly.xml" path="doc/members/member[@name='GetEnumsOnlyAsync(CancellationToken)']/*" />
-        public virtual async Task<Response<GetResponse6>> GetEnumsOnlyAsync(CancellationToken cancellationToken = default)
+        public virtual async Task<Response<GetResponse3>> GetEnumsOnlyAsync(CancellationToken cancellationToken = default)
```

Comment:
Why we have these changes?

Reply:
It's because of TCGC logic on naming anonymous model. Any code change could cause the names change. I've raised an issue before: https://github.com/Azure/typespec-azure/issues/87